app/poro/tour.rb

```diff
@@ -0,0 +1,6 @@
+class Tour
+
+  def navigation_link(coordinates)
+    NavigationService.new.get_tour(coordinates)
+  end
```

Comment:
Since this is used by all 3 classes, we decided to inherit from a parent class!

Reply:
pillar flex